### DISCUSSION TITLE
https://www.yunyitang.me/hugo-papermod-blog/

### DISCUSSION THREAD

**Discussion by giscus** [yunyit/yunyit.github.io#23] [Comments]
# [Hugo+PaperMod 双语博客搭建 Home-Info+Profile Mode](https://www.yunyitang.me/hugo-papermod-blog/)

从入门到踩坑到从坑里爬起来！ 安装 安装 Hugo：安装 Hugo 需要利用一个包管理工具。对于 MacOS 系统 Hugo 可以直接用 Homebrew 来安装，打开终端输入 brew install hugo 安装 Paper

https://www.yunyitang.me/hugo-papermod-blog/

**Comment by yunyit** [comment]
有劳捉虫，欢迎评论 👋🏼
